lib/Sema/ConstraintSystem.cpp

```diff
@@ -3528,10 +3528,26 @@ void ConstraintSystem::resolveOverload(ConstraintLocator *locator,
   if (isDebugMode()) {
     PrintOptions PO;
     PO.PrintTypesForDebugging = true;
-    llvm::errs().indent(solverState ? solverState->getCurrentIndent() : 2)
-      << "(overload set choice binding "
-      << boundType->getString(PO) << " := "
-      << adjustedRefType->getString(PO) << ")\n";
+
+    auto &log = llvm::errs();
```

Comment:
`getDebugLogger()` instead of `llvm::errs()`?

Reply:
`getDebugLogger` is on `SolverStep` in CSStep.cpp so it seems that I am unable to access it from ConstraintSystem.cpp